meilisearch/tests/auth/errors.rs

```diff
@@ -1,5 +1,5 @@
+use actix_web::http::StatusCode;
```

Comment:
So, if we import it from `actix_web`, do we still need to depend on `http` directly?